packages/grid/x-data-grid/src/components/cell/GridActionsCell.tsx

```diff
@@ -16,76 +17,198 @@ interface TouchRippleActions {
   stop: (event: any, callback?: () => void) => void;
 }
 
-type GridActionsCellProps = Pick<GridRenderCellParams, 'colDef' | 'id' | 'api' | 'hasFocus'> &
-  Pick<GridMenuProps, 'position'>;
+interface GridActionsCellProps extends Omit<GridRenderCellParams, 'value' | 'formattedValue'> {
+  value?: GridRenderCellParams['value'];
+  formattedValue?: GridRenderCellParams['formattedValue'];
+  position?: GridMenuProps['position'];
+}
 
 const GridActionsCell = (props: GridActionsCellProps) => {
+  const {
+    colDef,
+    id,
+    api,
+    hasFocus,
+    isEditable,
+    field,
+    value,
+    formattedValue,
+    row,
+    rowNode,
+    cellMode,
+    getValue,
+    tabIndex,
+    position = 'bottom-end',
```

Comment:
```suggestion
    field,
    row,
    rowNode,
    tabIndex,
    position = 'bottom-end',
```

Does it make sense to add props linked to cell values for the action cell?

Reply:
They come automatically from `GridCellParams`.

Reply:
I think I get your point. You are removing those properties to `other` to do the TODO comment